components/bootloader/Kconfig.projbuild

```diff
@@ -400,6 +400,26 @@ menu "Security features"
 
             Refer to https://docs.espressif.com/projects/esp-idf/en/latest/security/secure-boot.html before enabling.
 
+    config BOOT_SKIP_VALIDATE_OUT_OF_DEEP_SLEEP
+        bool "Skip image validation when exiting deep sleep"
+        depends on (SECURE_BOOT_ENABLED && SECURE_BOOT_INSECURE) || !SECURE_BOOT_ENABLED
+        select RTC_BOOTLOADER_RESERVED
```

Comment:
Seems to be a non-existent option?

Reply:
@igrr -

Good catch.  This should have been BOOTLOADER_RESERVE_RTC, not RTC_BOOTLOADER_RESERVED.  I changed the naming order to more closely match the other option similar within the linker script - CONFIG_BT_RESERVE_DRAM - but missed one spot.  Thanks.

- Tim